styleguide/containers/BaseContainer.jsx

```diff
@@ -13,9 +13,9 @@ export default class BaseContainer extends React.Component
         <p className="usa-font-lead">
           {this.props.summary}
         </p>
-        <div className="usa-grid">
```

Comment:
Yes - the `usa-grid` class adds padding on the left and right causing the contents to not align with the title and summary. It's not present on standards.usa.gov, so thought it was a mistake.